corporate/views/billing_page.py

```diff
@@ -39,6 +39,11 @@ def billing_home(
     user = request.user
     assert user.is_authenticated
 
+    # BUG: This should pass the acting_user; this is just working
+    # around that make_end_of_cycle_updates_if_needed doesn't do audit
+    # logging not using the session user properly.
+    billing_session = RealmBillingSession(user=None, realm=user.realm)
```

Comment:
@prakhar1144 can you put up a follow-up PR fixing this bug? I think the fix involves making `write_audit_log` support an `background_task` optional parameter that results in it not audit logging end-of-cycle updates as having an acting user.